exercises/.gitignore

```
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules
+videos
+
```

Add `screenshots`  to .gitignore
```suggestion
screenshots

```

---

see 15ca473c628b04b38df88975b08ca3c881f2091d